# Service which takes in:
#
#   full_path_to_file_load : full path file contain camera0~4.jpg
#   
# and returns:
#
#                 pose : the pose of the tag in the camera frame
#  tag_detection_image : an image with the detected tag's border highlighted and payload value printed

# The idea is to trigger the service, what is return is dosen't matter
string full_path_to_file_load
sensor_msgs/CameraInfo[] camera_info
---
apriltag_ros/AprilTagDetectionArray[] tag_detections